azurelinuxagent/common/osutil/ubuntu.py

```diff
@@ -100,7 +100,7 @@ def restart_if(self, ifname, retries=3, wait=5):
         """
         retry_limit=retries+1
         for attempt in range(1, retry_limit):
-            return_code=shellutil.run("systemctl restart systemd-networkd")
+            return_code=shellutil.run_command("systemctl restart systemd-networkd")
```

Comment:
Thanks for the change @taoyama. The usage of shellutil.run_command is different to that of shellutil.run, you can see an example in the implementation of restart_if.

I can make this change early next week if you don't get to it first. In the meanwhile I'll leave your PR open.

Reply:
Sorry, I have fixed and tested it. Could you review this again, @narrieta ?